### PR TITLE
UnicodeWriter character code to environment

### PR DESCRIPTION
**Pull request by yamamanx**
Hello.

Since I need to output CSV with CP932, I changed the character code to environment variable.

Best Regards.

### CODE REVIEW

File: redash/utils/__init__.py [hunk 1]
```diff
@@ -17,6 +18,7 @@
 from redash import settings
 
 COMMENTS_REGEX = re.compile("/\*.*?\*/")
+WRITER_ENCODE = os.environ.get('REDASH_WRITER_ENCODE', 'utf-8')
```
Comment (arikfr) [inline redash/utils/__init__.py]: Can you please change this to `WRITER_ENCODING` and `REDASH_CSV_WRITER_ENCODING`? Thanks.

**Comment by yamamanx** [comment]
Thanks @arikfr.
I changed them.

**Comment by arikfr** [comment]
Merged. Thanks.

**Comment by yamamanx** [comment]
Thanks

**Pull request merged**